ports/rp2/boards/W5100S_EVB_PICO/mpconfigboard.h

```diff
@@ -1,8 +1,9 @@
 // Board config for Wiznet W5100S-EVB-Pico.
 
 // Board and hardware specific configuration
-#define MICROPY_HW_BOARD_NAME             "W5100S-EVB-Pico"
-#define MICROPY_HW_FLASH_STORAGE_BYTES    (1408 * 1024)
+#define MICROPY_HW_BOARD_NAME               "W5100S-EVB-Pico"
+#define MICROPY_PY_NETWORK_DEFAULT_HOSTNAME "W5100S-EVB"
```

Comment:
I usually try not to mix HW and PY config options.  Maybe this can go down just below `MICROPY_PY_NETWORK`?